test/binCases/output/output-library-single/test.opts

```diff
@@ -0,0 +1,5 @@
+./index.js
+-o ../../../js/bin/output/output-library-single/bundle.js
```

Comment:
Don't think this is needed, what's the syntax in the other tests?

Reply:
removed. only the original output test used this flag, I just copied from there.